gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/ddl/GfxdDDLMessage.java

```diff
@@ -274,8 +274,10 @@ protected void processMessage(final DistributionManager dm)
       }
       return;
     }
-    SanityManager.DEBUG_PRINT(GfxdConstants.TRACE_DDLREPLAY, this.toString()
-        + " Starting execution");
+    if (!Misc.isSnappyHiveMetaTable(ddl.getCurrentSchema())) {
+      SanityManager.DEBUG_PRINT(GfxdConstants.TRACE_DDLREPLAY, this.toString()
+          + " Starting execution");
+    }
```

Comment:
After making the log level as WARN in postCreateDDLReplay and externalCatalog, I ran the cluster and found out the hive table related logs that were still getting logged and disabled them in this manner.